qa/tasks/cephfs/kernel_mount.py

```diff
@@ -66,7 +66,7 @@ def mount(self, mount_path=None, mount_fs_name=None, mountpoint=None):
                 '--',
                 self.mountpoint,
             ],
-            timeout=(5*60),
+            timeout=(5*60), check_status=False
         )
```

Comment:
I'm not sure about this -- the mount command could fail for reasons besides it being already mounted. Perhaps you should just check whether it's mounted before issuing the mount command? Or maybe consider unmounting and not checking the error and then redoing the mount?

Reply:
Yep, something like that would be better.

Reply:
On a proper look at the above snippet -

```
        self.client_remote.run(
            args=[
                'mkdir',
                '--',
                self.mountpoint,
            ],
            timeout=(5*60), check_status=False
)
```

Isn't this actually change fine? It just creates the mountpoint directory and doesn't mount anything. I recall, now, that this change was made because if the mountpoint directory is already created, the method would crash and the test would abort. I've disabled just this crash/abort part.

Reply:
Yes, I misread. I was thinking this was the mount command. For the mkdir it's  probably fine. That said, the usual way to deal with "create the directory if it doesn't already exist" is to use "mkdir -p".

Reply:
"mkdir -p" would be better.

Reply:
Made the change - https://github.com/ceph/ceph/pull/29421/files#diff-42931b13153c8474d563f8a82f789b51R55.

Thanks!